internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,35 @@
 package daysteps
 
 import (
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	s := strings.Split(data, ",")
+	if len(s) == 2 {
+		steps, err := strconv.Atoi(s[0])
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```